geomesa-spark/geomesa-spark-jts/src/test/scala/org/locationtech/geomesa/spark/BlankDataFrame.scala

```diff
@@ -0,0 +1,25 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+package org.locationtech.geomesa.spark
+
+import org.apache.spark.sql.{DataFrame, Row, SparkSession}
+import org.apache.spark.sql.types.StructType
+
+/**
+ * Mixin providing constructor for creating a DataFrame with a single row and no columns.
+ * Useful for testing the invocation of data constructing UDFs.
+ * @author sfitch 
```

Comment:
could you remove the author tag? we generally don't include those

Reply:
Sure... was auto generated by IDE.

FWIW, I had pasted the license file in there but I guess from @vpipkt 's comments you need spaces on either side?

Reply:
ah, yeah possibly - the licence check is pretty strict. You can run `mvn license:check` to make sure they're ok.
BTW, if you want to claim copyright, we can add another license template for you.

Reply:
Thanks; I/we appreciate that. In this case we should let it drop. Let's revisit if it arises again.